podcasts/DiscoverViewController.swift

```diff
@@ -190,7 +193,7 @@ class DiscoverViewController: PCViewController {
                 addSummaryController(CollectionSummaryViewController(), discoverItem: discoverItem)
             case ("network_list", _, _):
                 addSummaryController(NetworkSummaryViewController(), discoverItem: discoverItem)
-            case ("categories", _, _):
+            case ("categories", "category", _):
```

Comment:
It's to differentiate from the new "pills" summary style [added here](https://github.com/Automattic/pocket-casts-ios/pull/1602/files#diff-7ace00507c7ae070e7e49bb9884e38981c21f38977964a3ba9ace1adb32b26adR182). I didn't like that the order could matter here, so added the old summary style to be sure.

Do you think I should change it back just in case the summary style is missing or different somewhere and let the ordering decide?